mk/spksrc.service.start-stop-daemon

```diff
@@ -24,7 +24,7 @@ start_daemon ()
     fi
     if [ -n "${SERVICE_EXE}" ]; then
         if [ -n "${SERVICE_OPTIONS}" ]; then
-            start-stop-daemon -o -S -b -p ${PID_FILE} -m -c ${EFF_USER} -k 002 -x "${SERVICE_EXE}" -- "${SERVICE_OPTIONS}"
+            start-stop-daemon -o -S -b -p ${PID_FILE} -m -c ${EFF_USER} -k 002 -x "${SERVICE_EXE}" -- ${SERVICE_OPTIONS}
```

Comment:
@ymartin59 are there any side-effects for unquoting `${SERVICE_OPTIONS}` or is this a bugfix for all services htat use this script?

Reply:
Right now only syncthing is using this option but only with a single argument instead of multiple.

Reply:
I agree these quotes removal is a good idea.